src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/SQLIntegTestCase.java

```diff
@@ -103,6 +103,25 @@ protected Request getSqlRequest(String request, boolean explain) {
         return sqlRequest;
     }
 
+    protected String executeQuery(String query, String requestType) {
```

Comment:
We could make this method more generic by accepting a URL param list rather than `requestType`.

Reply:
I feel that this is an overkill as of now. When time will come, it can be refactored.

Reply:
It's okay. Thanks!